LDAP/ldap2s3/ldap2s3.yml.example

```diff
@@ -0,0 +1,18 @@
+ldap:
+    uri: "ldap.db.scl3.mozilla.com"
+    user: ""
+    password: ""
+    search_base:
+        groups: "ou=groups,dc=mozilla"
+        users: "dc=mozilla"
+    filter:
+        groups: "(objectClass=groupOfNames)"
+        users: "(&(objectClass=inetOrgPerson)(|(o:dn:=org)(o:dn:=com)(o:dn:=net)))"
```

Comment:
might want to add something like employee status filter here. its already done by LDAP, but doesn't hurt?

Reply:
"(&(!(employeeType=DISABLED))(objectClass=inetOrgPerson)(|(o:dn:=org)(o:dn:=com)(o:dn:=net)))"

This should help keep things consistent in case we open up ACLs to allow this bind user to "see" disabled users, adding this to the filter will still filter them out.

Reply:
added tested works :)